### ISSUE TITLE
iq-tree crashes with signal aborted

### ISSUE THREAD

**Issue by matt-sd-watson** [iqtree/iqtree2#26]
Hello. 

I am running iqtree2 with fconst on a set of ncov19 genomes processed through the ivar pipeline. I have iqtree2 set to run through a shell script with a bootstapping size of 1000. When executing through the shell script, I receive the following error message: 

![iqtree2_error_signal_aborted](https://user-images.githubusercontent.com/40243147/117330162-67b76680-ae63-11eb-8ce7-81ca6164bde4.png)

The error does not occur when taking the exact commands out of the shell script. Is there a way to specify these commands better in the sh file to prevent a signal abort? 

Constant site positions are determined through snp-sites with the following commands in the script: 

```
coor=$(snp-sites -C 618_aln.fa)
iqtree2 -mset HKY,TIM2,GTR -mfreq F -mrate G,R -alrt 1000 -bb 1000 -fconst $coor -pre 618 -o MN908947 -s 618_aln_VarsOnly.fasta
```

**Comment by bqminh** [comment]
Can you please attach the full log file? The screenshot is not very informative.

**Comment by matt-sd-watson** [comment]
The full log file for the run is posted here. Some information is de-identified for privacy reasons. 
The log suggests that a sequence failed composition, but when running the same command outside of the shell script, the command succeeds and the FASTA is generated. 

[617_aln_VarsOnly.fasta.log](https://github.com/iqtree/iqtree2/files/6441880/617_aln_VarsOnly.fasta.log)

**Comment by bqminh** [comment]
Your -fconst option is not correct. It should provide a list of hour const pattern frequencies for A, C, G and T.

So it should be -fconst fA,fC,fG,fT

note that it is a comma separated list without empty space.

Minh

**Comment by matt-sd-watson** [comment]
Hello Minh, 

Thank you for your input. the -fconst parameter was passed as a shell variable so its values were correct, however the version of snp-sites in my activated conda environment did not have the -C parameter available for the constant sites, hence why it ran correctly when I deactivated the environment. I have made the necessary version upgrades to avoid this and now it runs properly in the shell. thank you for your help.
